_layouts/home.html

```diff
@@ -32,7 +32,7 @@
 
         {% include google-tag-manager.html %}
 
-        <a href="https://github.com/gocd/gocd/">
+        <a href="https://github.com/gocd/gocd/" analytics-label="github-fork-me">
```

Comment:
Question about these kind of labels on external links: I presume that the link will be prevented from being traversed to, while the analytics information is sent. Is this right? If so, for how long? What if journey is down? If not, how do you ensure a ping to the analytics backend before the link is clicked and the user is sent away to a different page?